honda_civic_touring_2016_can.dbc

```diff
@@ -274,6 +274,7 @@ BO_ 806 SCM_FEEDBACK: 8 SCM
  SG_ MAIN_ON : 28|1@0+ (1,0) [0|1] "" NEO
  SG_ RIGHT_BLINKER : 27|1@0+ (1,0) [0|1] "" NEO
  SG_ LEFT_BLINKER : 26|1@0+ (1,0) [0|1] "" NEO
+ SG_ REVERSE_LIGHT : 18|1@0+ (1,0) [0|1] "" NEO
  SG_ COUNTER : 61|2@0+ (1,0) [0|3] "" NEO
  SG_ CHECKSUM : 59|4@0+ (1,0) [0|3] "" NEO
 

```

Comment:
How do you know this is reverse lights and not reverse gear engaged?

Reply:
Just a theory since this in on a message with other indicator lights and not a message for gearbox. I don't currently have a set up to set messages to the bus to confirm though

Reply:
The msg comes from steering column, which is where buttons and turns levers are. Seems strange it sends info about reverse gear or lights. Anyway, if you saw the correlation between reverse gear it's sufficient and better than nothing. Can you just add a clarification comment at the end of the dbc to highlight the uncertainty?

Reply:
Added